generators/generator-constants.js

```diff
@@ -94,7 +94,7 @@ const LANGUAGES = [
     },
     { name: 'Armenian', dispName: 'Հայերեն', value: 'hy' },
     { name: 'Catalan', dispName: 'Català', value: 'ca' },
-    { name: 'Chinese (Simplified)', dispName: '中文（简体）', value: 'zh-cn' },
+    { name: 'Chinese (Simplified)', dispName: '中文（简体）', value: 'zh-CN' },
```

Comment:
I dont think this line should be changed. This will cause problem when people upgrade their application.

Reply:
yes , i seen client have a i18n floder.

Reply:
please revert this line

Reply:
Also please revert all file renames to uppercase

Reply:
so  modify   this line   https://github.com/jhipster/generator-jhipster/blob/efc1e35ef1c0dae087c613760cf2c475447349d7/generators/generator-base-private.js#L91 
ok?

Reply:
this is  JDK 8 and JRE 8 Supported Locales
http://www.oracle.com/technetwork/java/javase/java8locales-2095355.html

Reply:
No. Why are you changing this in the first place?

Reply:
so we can do a transfer,ok ?